packages/extension-ui/src/components/Dropdown.tsx

```diff
@@ -30,6 +30,8 @@ interface Props extends ThemeProps {
   value?: string;
 }
 
+const ALEPH_ZERO_GENESIS_HASH = '0x70255b4d28de0fc4e1a193d7e175ad1ccef431598211c55538f1018651a0344e';
+
```

Comment:
I would move it to an external file with some config configuration rather than part of `Dropdown` component :)